### ISSUE TITLE
How to use gridTemplateAreas?

### ISSUE THREAD

**Issue by Unmoving** [japgolly/scalacss#117]
No matter how hard I try, I always get:

```
grid-template-areas: 
      &quot;header header&quot;
      &quot;main sidebar&quot;
      &quot;footer footer&quot;
    ;
```

in the .css file.

```
val areas =
     """
       "header header"
       "main sidebar"
       "footer footer"
     """
```

would print to 

```
     "header header"
     "main sidebar"
     "footer footer"
```

but after doing this: `gridTemplateAreas := areas` the 

> "
are replaced by 
> &quot;
Can I somehow prevent this? There is no doc for this, sadly... probably just too noobish.

The workaround I am using right now is

```
val areas =
    """
      'header header'
      'main sidebar'
      'footer footer'
    """
```

Which generates

```
grid-template-areas: 
      ''header header''
      ''main sidebar''
      ''footer footer''
    ;
```

So instead of a double quotation mark, TWO single ones!

**Comment by japgolly** [comment]
Thanks for reporting. This is indeed a bug.

**Comment by japgolly** [comment]
Fixed in 49f233d1699bcb8f615517fff69553abe6d00514
